easybuild/easyconfigs/g/goolf/goolf-1.6.20.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = "Toolchain"
+
+name = 'goolf'
+version = '1.6.20'
```

Comment:
bumped OpenMPI from 1.7.x (in goolf 1.6.10) to 1.8.x, so make toolchain version 1.7.x here, e.g. 1.7.12 (or 1.7.14, something like that)